specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2024-09-10-preview/HybridCompute.json

```diff
@@ -2771,6 +2982,137 @@
       },
       "description": "Describes Extension Metadata properties"
     },
+    "ExtensionValueListResultV2": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/ExtensionValueV2"
+          },
+          "description": "The list of V2 extension metadata.",
+          "x-ms-identifiers": []
+        }
+      },
+      "description": "The List Extension Metadata V2 response."
+    },
+    "ExtensionValueV2": {
+      "type": "object",
+      "properties": {
+        "extensionUris": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "type": "string"
+          },
+          "description": "A list of locations where the extension packages can be found.",
+          "x-ms-identifiers": []
+        },
+        "extensionSignatureUri": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Location of the signature files for the extension."
+        },
+        "extensionPackageSizeMb": {
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true,
+          "description": "The size of extension package in megabytes."
+        },
+        "operatingSystem": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Operating system that can support the extension."
+        },
+        "computeRole": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Compute role that can support the extension. IaaS, PaaS or All."
+        },
+        "architecture": {
```

Comment:
>architecture [](http://example.com/codeflow?start=9&length=12)

can this also be converted into an enum? if not can you add in the description what format the values should be

Reply:
Similar question/consideration here.